test/server/cards/01-Core/EpicQuest.spec.js

```diff
@@ -0,0 +1,50 @@
+describe('Epic Quest', function() {
+    integration(function() {
+        describe('Epic Quest\'s ability', function() {
+            beforeEach(function() {
+                this.setupTest({
+                    player1: {
+                        house: 'sanctum',
+                        inPlay: ['epic-quest'],
+                        hand: ['sequis', 'clear-mind', 'glorious-few', 'virtuous-works', 'gorm-of-omm', 'round-table', 'protectrix']
+                    },
+                    player2: {
+                        amber: 1,
+                        inPlay: ['troll', 'gauntlet-of-command']
+                    }
+                });
+            });
+
+            it('should not forge a key if played 6 or less', function() {
+                this.player1.play(this.sequis);
+                this.player1.play(this.clearMind);
+                this.player1.play(this.gloriousFew);
+                this.player1.play(this.virtuousWorks);
+                this.player1.play(this.protectrix);
+                this.player1.play(this.gormOfOmm);
+                expect(this.player1.amber).toBe(4);
+                this.player1.useAction(this.epicQuest, true);
+                expect(this.epicQuest.location).toBe('play area');
+                expect(this.epicQuest.exhausted).toBe(true);
+                expect(this.player1).toHavePrompt('Choose a card to play, discard or use');
+            });
+
+            it('should not forge a key if played 7', function() {
```

Comment:
Hey just a heads up - I think you meant `should forge a key` instead of `should not`.

Reply:
I will fix, thanks!